src/SFA.DAS.ApplyService.Application/Apply/UploadPageAnswersResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.ApplyService.Application.Apply
+{
+    public class UploadPageAnswersResult
+    {
+        public bool ValidationPassed { get; set; }
+        public List<KeyValuePair<string, string>> ValidationErrors { get; set; }
+        public string NextAction { get; set; }
+        public string NextActionId { get; set; }
+    }
```

Comment:
You can use `SetPageAnswersResponse` from `SFA.DAS.QnA.Api.Types` instead of creating this one.